osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -76,11 +77,12 @@ public bool Ripple
 
         public OsuLogo()
         {
+            AlwaysPresent = true;

```

Comment:
Honestly, we should just add comments wherever this is used, because this still keeps the elements drawn, and in this case - draws to a framebuffer and breaks batching. What's the use case for AlwaysPresent here?